docs/generators/fsharp-giraffe-server.md

```diff
@@ -15,7 +15,7 @@ sidebar_label: fsharp-giraffe-server
 |packageName|F# module name (convention: Title.Case).| |OpenAPI|
 |packageVersion|F# package version.| |1.0.0|
 |packageGuid|The GUID that will be associated with the C# project| |null|
-|sourceFolder|source folder for generated code| |OpenAPI/src|
+|sourceFolder|source folder for generated code| |OpenAPI\src|
```

Comment:
shouldn't this be `OpenAPI/src`

Reply:
Was auto-generated, not my change. I can revert it back in needed

Reply:
Seems to make the test fail... will revert that file